client/utils/utils.go

```diff
@@ -115,7 +122,7 @@ func PrintUnsignedStdTx(w io.Writer, txBldr authtxb.TxBuilder, cliCtx context.CL
 	}
 	json, err := cliCtx.Codec.MarshalJSON(stdTx)
 	if err == nil {
-		fmt.Fprintf(w, "%s\n", json)
+		fmt.Fprintf(cliCtx.Output, "%s\n", json)
```

Comment:
Since we are here, can we introduce the `CLIContext.Diagnostics`/`CLIContext.Error` stream?

Reply:
I don't want to tackle that as part of this PR. Can you open an issue for this work?

Reply:
Yeah forget it. We can do it in a separate PR :+1: